modules/core/core-repo/src/main/java/com/enonic/xp/repo/impl/node/CheckNodeExistsCommand.java

```diff
@@ -37,20 +39,21 @@ public static Builder create( AbstractNodeCommand source )
 
     public boolean execute()
     {
-        final InternalContext context = InternalContext.from( ContextAccessor.current() );
+        final InternalContext context =
```

Comment:
Does it mean we always check existence on primary? (I actually like it, but it may hurt performance of Load/Import)